server/src/dtos/user-preferences.dto.ts

```diff
@@ -103,6 +106,8 @@ class EmailNotificationsResponse {
 class DownloadResponse {
   @ApiProperty({ type: 'integer' })
   archiveSize!: number;
+
+  includeEmbeddedVideos: boolean = false;
```

Comment:
Do we need to patch this in mobile client?

Reply:
I don't think so. It is a new property and it has a default value.